components/12-argo-events/webhook-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: selfsigned-cluster-issuer
+    nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+  name: nautobot-webhook
+spec:
+  ingressClassName: nginx
+  rules:
+  - host: events.local
+    http:
+      paths:
+      - path: /nautobot        # must match endpoint defined in EventSource
```

Comment:
I don't think you need the ingress here. Nautobot is running within the cluster and should just use the service that the event source created.

Reply:
Agreed. I have found the need to receive remote webhooks in development, though we can abstract ingress from this repo.